tests/beaker/tests/aix/create_lv_with_param_type.rb

```diff
@@ -0,0 +1,49 @@
+require 'master_manipulator'
+require 'lvm_helper'
+require 'securerandom'
+
+test_name "FM-4969 - C97368 - create logical volume with param 'type' aix"
```

Comment:
Test name is misleading. The type is 'jfs' and not 'aix'. I would reword the title to indicate that 'jfs' is AIX specific.